Sources/Purchasing/StoreKit1/ProductsFetcherSK1.swift

```diff
@@ -186,9 +186,15 @@ extension ProductsFetcherSK1: SKProductsRequestDelegate {
         }
     }
 
-    func clearCache() {
+    /// - Returns (via callback): The product identifiers that were removed, or empty if there were not
+    ///   cached products.
+    func clearCache(completion: @escaping ((Set<String>) -> Void)) {
```

Comment:
Maybe provide a default value for the callback?
```suggestion
    func clearCache(completion: @escaping ((Set<String>) -> Void)? = nil) {
```